### ISSUE TITLE
NullPointerException if schema is missing from content

### ISSUE THREAD

**Issue by msnijder30** [quen2404/openapi-diff#58]
A `NullPointerException` is thrown if the `schema` is missing from the `content`

### The files
I have included a zip with two files, `broken-openapi.yml` which throws the nullpointer and `working-openapi.yml` which works as expected.

[openapi-examples.zip](https://github.com/quen2404/openapi-diff/files/2495009/openapi-examples.zip)

### What's different?
The difference between the files is that the following part in `working-openapi.yml` :
```xml
  content:
    application/json:
      schema:
        $ref: '#/components/schemas/User'
```

is replaced with the following:
```xml
  content:
    text/plain: {}
```

### Reproduction & error
The code which I've used to test this is as follows:
```java
public static void main(String[] args) {
    ChangedOpenApi diff =
            OpenApiCompare.fromLocations("broken-openapi.yml", "broken-openapi.yml");
}
```

A nullpointer is thrown, see the stacktrace below:
```java
Exception in thread "main" java.lang.NullPointerException
	at com.qdesrame.openapi.diff.compare.SchemaDiff.diff(SchemaDiff.java:62)
	at com.qdesrame.openapi.diff.compare.ContentDiff.diff(ContentDiff.java:35)
	at com.qdesrame.openapi.diff.compare.ResponseDiff.computeDiff(ResponseDiff.java:39)
	at com.qdesrame.openapi.diff.compare.ResponseDiff.computeDiff(ResponseDiff.java:16)
	at com.qdesrame.openapi.diff.compare.ReferenceDiffCache.cachedDiff(ReferenceDiffCache.java:54)
	at com.qdesrame.openapi.diff.compare.ResponseDiff.diff(ResponseDiff.java:29)
	at com.qdesrame.openapi.diff.compare.ApiResponseDiff.diff(ApiResponseDiff.java:32)
	at com.qdesrame.openapi.diff.compare.OperationDiff.diff(OperationDiff.java:41)
	at com.qdesrame.openapi.diff.compare.PathDiff.diff(PathDiff.java:31)
	at com.qdesrame.openapi.diff.compare.PathsDiff.lambda$diff$2(PathsDiff.java:53)
	at java.util.LinkedHashMap$LinkedKeySet.forEach(LinkedHashMap.java:559)
	at com.qdesrame.openapi.diff.compare.PathsDiff.diff(PathsDiff.java:36)
	at com.qdesrame.openapi.diff.compare.OpenApiDiff.compare(OpenApiDiff.java:92)
	at com.qdesrame.openapi.diff.compare.OpenApiDiff.compare(OpenApiDiff.java:66)
	at com.qdesrame.openapi.diff.OpenApiCompare.fromSpecifications(OpenApiCompare.java:99)
	at com.qdesrame.openapi.diff.OpenApiCompare.fromLocations(OpenApiCompare.java:88)
	at com.qdesrame.openapi.diff.OpenApiCompare.fromLocations(OpenApiCompare.java:76)
	at com.github.msnijder30.openapidifftest.Main.main(Main.java:19)
```

### Question

Is there a way this could be fixed? A schema of `{}` in OpenAPI is valid, the documents provided in this issue are generated by [swagger-core](https://github.com/swagger-api/swagger-core) itself.

**Comment by msnijder30** [comment]
Looking into the code the assumption is made in `SchemaDiff#diff` that `Schema left` and `Schema right` are not null, which in my case they are.

**Comment by quen2404** [comment]
Hello @msnijder30 ! Thank you for this report, and thank you too for the PR. 
I will try to reproduce your problem

**Comment by msnijder30** [comment]
Sure, let me know if you have any questions.

**Comment by quen2404** [comment]
Fixed with commit [313b110](https://github.com/quen2404/openapi-diff/commit/313b1100b80a46f6281911640a3ae1b270f2d07f)
